doajtest/unit/test_api_crud_application.py

```diff
@@ -408,7 +408,7 @@ def test_05_outgoing_application_do(self):
         oa = OutgoingApplication()
 
         # make one from an incoming application model fixture
```

Comment:
It's still unclear to me why this has been renamed in so many places.

Reply:
Because these tests assert if `current_journal is not None` and this is true only for update requests (that's the difference between the application and update request.

Reply:
cool, I'm happy with that